cmd/soci/commands/index/info.go

```diff
@@ -32,12 +34,24 @@ import (
 	"github.com/urfave/cli"
 )
 
+var (
+	defaultIndexInfoTimeout = 10 * time.Second
```

Comment:
It feels weird to set a default timeout for just `index info`. `internal.AppContext` already sets the timeout on the context if it's > 0. If we want a default, that should be on the flag itself, but I don't  think we do.

Reply:
Okay yea, should be fine to have no timeout, and let users set it if they are experiencing issues.